fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/MetricMaker.java

```diff
@@ -131,41 +131,24 @@ public LogicalMetric make(String metricName, String dependentMetric) {
     protected abstract int getDependentMetricsRequired();
 
     /**
-     * Fetch the TemplateDruidQuery of the dependent metric from the Metric Dictionary.
-     *
-     * @param name  Name of the metric to fetch the template druid query from
-     * @return The template druid query of the metric
-     */
-    protected TemplateDruidQuery getDependentQuery(String name) {
```

Comment:
We can't just remove this, since it was `protected` and in a class that we _intended_ to be extended. We need to `@Deprecate` it instead.